### DISCUSSION TITLE
v2.1.0 - Release

### DISCUSSION THREAD

**Discussion by Wenn0101** [sparkfun/Arduino_Apollo3#393] [General]
-Added pull-up to RX pins by default for UART configs
-Defined missing Wire1 for thing plus
-RTC low power alarm examples
-Watchdog timer implementation
-Default Baud rate for SVL on macOS and linux fail
-Fixed - Example MicrophoneOutput not compiling Artemis Thing Plus
-Fixed - PDM Library compilation errors
-Fixed - Serial1 Doesn’t compile for the Artemis Nano board
-Fixed - Interrupts don’t fire on change
-SPI mode fixes
-Fix bug that prevents directly changing PWM frequency
-Add burst mode
-Reduce release package size

<hr /><em>This discussion was created from the release <a href='https://github.com/sparkfun/Arduino_Apollo3/releases/tag/v2.1.0'>v2.1.0 - Release</a>.</em>

**Comment by adamgarbo** [comment]
Thanks @Wenn0101! Great to see all the fixes.

**Reply by Wenn0101** [reply]
Thanks for your contributions to this release!

v1.2.2 is coming next i'll bring in your other PR and probably a couple other small fixes.

**Comment by paulvha** [comment]
Received the update today in Arduino IDE. The update went without issues. However, when downgrading again to 2.0.6 you get the known problem https://github.com/sparkfun/Arduino_Apollo3/issues/377 as the name of the tar-file is the same. The problem does not happen when you update to 2.1.0 as the tar-file is (much :-)) smaller. 

The downloaded the tar-file is 173,1MB, extracting that and then creating a new tar-file (same extensions) gives a size of 101.8Mb (Ubuntu 20.4). So you can make it even smaller with a different compress utility.. AND by removing the MBED-OS/targets-folders for different boards than Apollo3, which are NOT used at all you save another 513MB (or about 25 MB in the tar-file). That way the tar-file would 'only' be ~75MB (instead of 173Mb) and the real board library footprint is reduced to ~232MB instead of ~746MB.

Last point (cosmetic) there is an empty Arduino_Apollo3.tar.gz in the boards-library installed with V 2.1.0. 

regards,
Paul

**Reply by Wenn0101** [reply]
I will try to remove the unwanted targets, and maybe some unused features in future releases.

where is that empty Arduino_Apollo3.tar.gz you mentioned? I don't see it :D

**Comment by paulvha** [comment]
thanks
The empty tar is in root directory of the boards library:   .arduino15/packages/SparkFun/hardware/apollo3/2.1.0/Arduino_Apollo3.tar.gz

Regards,
Paul

**Reply by Wenn0101** [reply]
OH sure enough there it is, how did that get there :flushed:.
I'll make sure not to include that in the next one!

**Reply by paulvha** [reply]
hahaha .. it is only cosmetic..
